setup.py

```diff
@@ -87,9 +87,11 @@ def finalize_options(self):
             if SYSTEM_FITSIO_LIBDIR is not None:
                 self.library_dirs.insert(0, SYSTEM_FITSIO_LIBDIR)
         else:
-            # We defer configuration of the bundled cfitsio to build_extensions
-            # because we will know the compiler there.
-            self.include_dirs.insert(0, self.cfitsio_build_dir)
+            # we put our include dir first to avoid other fitsio install's
+            # header files
+            self.include_dirs.insert(
+                0, os.path.join(".", self.cfitsio_build_dir)
+            )
```

Comment:
```suggestion
            # We defer configuration of the bundled cfitsio to build_extensions
            # because we will know the compiler there.
            self.include_dirs.insert(0, self.cfitsio_build_dir)
```